core/opendaq/opendaq/include/opendaq/instance.h

```diff
@@ -22,6 +22,8 @@
 
 BEGIN_NAMESPACE_OPENDAQ
 
+struct IInstanceBuilder;
+
 /*!
  * @ingroup opendaq_devices
```

Comment:
Should we remove the `setRootDevice` call from IInstance?

Reply:
I think we can do that